components/AudioPlayer/AudioPlayer.tsx

```diff
@@ -201,6 +202,7 @@ export const AudioPlayer = ({
               </Box>
               <Box className={classes.volumeControls}>
                 <IconButton
+                  aria-label="Toggle Mute"
```

Comment:
This should probably update based on player state: `aria-label={muted ? 'Mute' : 'Unmute'}`

Reply:
Fixed!